frontend/src/components/v2/SecretInput/SecretInput.tsx

```diff
@@ -1,10 +1,15 @@
 /* eslint-disable react/no-danger */
-import { forwardRef, TextareaHTMLAttributes } from "react";
+import React, { forwardRef, TextareaHTMLAttributes, useRef, useState } from "react";
 import { twMerge } from "tailwind-merge";
 
+import {
```

Comment:
Let's keep this as a different component. Secret input is used in multiple places where secret reference is not needed. May be name the new component as `InfisicalSecretInput`. We can rename this component later